packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -78,7 +78,7 @@ const EntryValidationText = ({ errors, isPublished }) => {
         <Typography textColor="success600" fontWeight="bold">
           {formatMessage({
             id: 'app.utils.published',
-            defaultMessage: 'Published',
+            defaultMessage: 'Already Published',
```

Comment:
What do you think about changing the id here too? This is not really a general translation anymore and maybe should live in the content-manager translations instead?

Reply:
ok, I created a new ID let me know if it works for you